lib/ruby_lsp/setup_bundler.rb

```diff
@@ -402,5 +403,19 @@ def base_bundle_command(env)
 
       "bundle"
     end
+
+    sig { void }
+    def print_progress_to_stderr!
+      Bundler::Thor::Shell::Basic.prepend(Module.new do
```

Comment:
Are we confident this will work in older Bundler versions? (e.g. has this part of Bundler been stable for a long time?)